ui/src/views/admin/settings/SettingsView.vue

```diff
@@ -133,13 +134,15 @@ export default {
       return this.userData.email;
     },
     isPasswordLengthOkay() {
-      return this.formData.password.length >= 8;
+      return this.formData.new_password.length >= 8;
     }
   },
   methods: {
     onFormSubmit() {
       // to do: post form data to back-end's update admin api
-      this.$notify.toast("New settings have been saved!");
+      adminChangePassword(this.formData).then((response) => {
```

Comment:
This way, you are also sending the email; even though its not needed (the body req only needed the new_password). I already fixed the one in user/settings XD  Rather than using `this.formData` directly, you can assign the value to a new Object and send it instead.

Reply:
ok, I have refactored it so that only necessary data (i.e. only new pwd) is sent to api. XD